java/arcs/android/storage/service/IStorageServiceCallback.aidl

```diff
@@ -17,5 +17,5 @@ import arcs.android.storage.service.IResultCallback;
 /** Variant of ProxyCallback intended for StorageService communications. */
 interface IStorageServiceCallback {
     /** Handles an incoming ProxyMessage. */
-    void onProxyMessage(in ParcelableProxyMessage message, IResultCallback resultCallback);
+    void onProxyMessage(in ParcelableProxyMessage message);
```

Comment:
Shouldn't you add the 'oneway' keyword?
"The oneway keyword modifies the behavior of remote calls. When used, a remote call does not block; it simply sends the transaction data and immediately returns. The implementation of the interface eventually receives this as a regular call from the Binder thread pool as a normal remote call. If oneway is used with a local call, there is no impact and the call is still synchronous."

Reply:
Yes you're right, I think that makes sense here. 

Do you think it makes sense to apply `oneway` to the other methods in the interface which are `void` and provide a callback for getting a response?

Reply:
I don't know the underlying Android implications (ian may know), but anything that tells AIDL/Binder "don't bother waiting, fire and forget" seems like a good idea to me.